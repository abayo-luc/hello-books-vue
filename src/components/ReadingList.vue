<template>
  <div class="reading-list">
    <list-header />
    <reading-list-card />
  </div>
</template>
<script>
import ListHeader from './headers/ListHeader.vue';
import ReadingListCard from './cards/ReadingListCard.vue';

export default {
  name: 'ReadingList',
  components: {
    ListHeader,
    ReadingListCard,
  },
};
</script>
<style lang="scss" scoped>
.reading-list {
  position: relative;
  width: 100%;
}
</style>
