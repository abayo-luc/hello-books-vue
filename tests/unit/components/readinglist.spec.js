import {
  mount,
} from '@vue/test-utils';
import ReadingList from '../../../src/components/ReadingList.vue';

const wrapper = mount(ReadingList);
describe('ReadingList.vue', () => {
  it('should render reading list', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
