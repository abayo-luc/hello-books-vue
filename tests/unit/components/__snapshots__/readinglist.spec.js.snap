// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReadingList.vue should render reading list 1`] = `
<div class="reading-list">
  <div class="reading-list-header">
    <h3>My Current List</h3>
  </div>
  <div class="reading-list-card dark-card">
    <div class="single-item"><img src="../../assets/icons/open-book.svg" class="icon">
      <div class="book-details">
        <p class="book-name">Things Falll Apart</p>
        <p class="book-pages"><span>Page № :</span>1232
        </p>
      </div>
    </div>
    <div class="single-item"><img src="../../assets/icons/open-book.svg" class="icon">
      <div class="book-details">
        <p class="book-name">Things Falll Apart</p>
        <p class="book-pages"><span>Page № :</span>1232
        </p>
      </div>
    </div>
    <div class="single-item"><img src="../../assets/icons/open-book.svg" class="icon">
      <div class="book-details">
        <p class="book-name">Things Falll Apart</p>
        <p class="book-pages"><span>Page № :</span>1232
        </p>
      </div>
    </div>
    <div class="single-item"><img src="../../assets/icons/open-book.svg" class="icon">
      <div class="book-details">
        <p class="book-name">Things Falll Apart</p>
        <p class="book-pages"><span>Page № :</span>1232
        </p>
      </div>
    </div>
    <div class="single-item"><img src="../../assets/icons/open-book.svg" class="icon">
      <div class="book-details">
        <p class="book-name">Things Falll Apart</p>
        <p class="book-pages"><span>Page № :</span>1232
        </p>
      </div>
    </div>
  </div>
</div>
`;
